crates/bevy_render/src/camera.rs

```diff
@@ -194,6 +194,7 @@ impl NormalizedRenderTargetExt for NormalizedRenderTarget {
             NormalizedRenderTarget::TextureView(id) => {
                 manual_texture_views.get(id).map(|tex| &tex.texture_view)
             }
+            NormalizedRenderTarget::None { .. } => None,
```

Comment:
I'm somewhat surprised this "just works" but is really nice to see. No `ViewTarget` means the nodes don't get run, cool!